src/website/src/app/documentation/demos/wizard/wizard.demo.html

```diff
@@ -701,7 +701,7 @@ <h5>The wizard</h5>
     ...
 &#125;)
 export class WizardJumpToDemo &#123;
-    @ViewChild("ohai", {static: false}) wizard: ClrWizard; // makes wizard accessible through this.wizard!!!
+    @ViewChild("ohai") wizard: ClrWizard; // makes wizard accessible through this.wizard!!!
```

Comment:
Is the static still needed for the docs? I saw that it was causing the site build to fail because it needed to be escaped.

Reply:
I'm removing the docs updates for now. Rather than escaping them I'd rather port them over to the `const` pattern that we now use. 

wrt replacing Gemini - Once I get Angular8 certified and v2 release I'll update the Aplitools branch: https://github.com/vmware/clarity/tree/topic/applitools and get that ready to merge into master so that we can replace gemini with that.